errbot/backends/test.py

```diff
@@ -454,7 +454,10 @@ def assertCommand(self, command, response, timeout=5):
         """Assert the given command returns the given response"""
         self.bot.push_message(command)
         msg = self.bot.pop_message(timeout)
-        assert response in msg
+        try:
+            assert response in msg
+        except AssertionError:
+            raise AssertionError("'{}' not in '{}'".format(response, msg))
```

Comment:
As this is just going to catch the exception raised by the assert statement, it's more clear and concise to write this simply as:

```python
if response not in msg:
    raise AssertionError("'{}' not in '{}'".format(response, msg))
```

Alternatively, the following should also work (I think I like this best actually):

```python
assert response in msg, "'{}' not in '{}'".format(response, msg)
```